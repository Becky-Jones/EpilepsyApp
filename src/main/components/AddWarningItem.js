import React, { useState } from "react";
import { StyleSheet, View, Text, TextInput, Button } from "react-native";
const styles = require("../pages/stylesheets/styles");

export default function addWarningItem(props) {
  const [item, setItem] = useState("");
  return (
    <View>
      <TextInput
        onChangeText={(textVal) => {
          setItem(textVal);
        }}
        style={styles.input}
        placeholder="Warning in format. 1:00 2:00"
        placeholderTextColor="white"
      ></TextInput>
      <Button
        style={thispage.btn}
        onPress={() => {
          props.addWarningItem(item);
        }}
        title="Add warning"
      ></Button>
    </View>
  );
}

const thispage = StyleSheet.create({
  view: {
    width: 350,
    height: 500,
    backgroundColor: "lightgrey",
    margin: 10,
    padding: 8,
    color: "white",
    borderRadius: 14,
    fontSize: 18,
    fontWeight: "500",
  },
  textBox: {
    backgroundColor: "white",
    width: 350,
    height: 30,
    margin: 10,
    padding: 8,
    color: "black",
    borderRadius: 14,
    fontSize: 12,
    fontWeight: "500",
  },
  textInput: {
    color: "white",
    fontSize: 16,
    fontWeight: "500",
  },
  View: {
    width: 350,
    margin: 10,
    backgroundColor: "lightgrey",
    borderRadius: 14,
  },
  btn: {
    width: 200,
    borderRadius: 14,
    color: "red",
  },
});
