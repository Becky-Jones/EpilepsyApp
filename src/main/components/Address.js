export class Address {
  constructor(address, city, postcode) {
    this.address = address;
    this.city = city;
    this.postcode = postcode;
  }

  /*****************************
   * GETTERS
   *****************************/

  getAddressLine() {
    return this.address;
  }

  getCity() {
    return this.city;
  }

  getPostcode() {
    return this.postcode;
  }

  getFullAddress() {
    return this.address + ", " + this.city + ", " + this.postcode;
  }

  /*****************************
   * SETTERS
   *****************************/

  setAddress(address) {
    this.address = address;
  }

  setCity(city) {
    this.city = city;
  }

  setPostcode(postcode) {
    this.postcode = postcode;
  }
}
