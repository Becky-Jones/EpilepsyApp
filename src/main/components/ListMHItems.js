import React from "react";
import { View, Text, TouchableOpacity, FlatList, Button, StyleSheet } from "react-native";
const styles = require("../pages/stylesheets/styles");

export default function ListMHItems(props) {
  return (
    <View  style={thispage.view}>
      <FlatList
        data={props.listMHItems}
        renderItem={({ item }) => (
          <View>
            <TouchableOpacity
              style={{
                padding: 5,
                backgroundColor: "white",
                borderBottomWidth: 1,
                borderColor: "#eee",
                width: 300,
                marginTop: 5,
                justifyContent: "space-between",
                flexDirection: "row",
                display: "flex",
              }}
            >
              <Text> {item.impact}</Text>
              <Button
                title="Delete"
                onPress={() => {
                  props.deleteMHItem(item.id);
                }}
              ></Button>
            </TouchableOpacity>
          </View>
        )}
      ></FlatList>
    </View>
  );
}

const thispage = StyleSheet.create({
  view: {
    width: 350,
    height: 350,
    backgroundColor: "lightgrey",
    margin: 10,
    padding: 8,
    color: "white",
    borderRadius: 14,
    fontSize: 18,
    fontWeight: "500",
  },
});
