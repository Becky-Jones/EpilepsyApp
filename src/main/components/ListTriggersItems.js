import React from "react";
import { View, Text, TouchableOpacity, FlatList, Button, StyleSheet } from "react-native";
const styles = require("../pages/stylesheets/styles");

export default function ListTriggersItems(props) {
  return (
    <View  style={thispage.view}>
      <FlatList
        data={props.listTriggersItems}
        renderItem={({ item }) => (
          <View>
            <TouchableOpacity
              style={{
                padding: 5,
                backgroundColor: "white",
                borderBottomWidth: 1,
                borderColor: "#eee",
                width: 300,
                marginTop: 5,
                justifyContent: "space-between",
                flexDirection: "row",
                display: "flex",
              }}
            >
              <Text> {item.trigger}</Text>
              <Button
                title="Delete"
                onPress={() => {
                  props.deleteTriggersItem(item.id);
                }}
              ></Button>
            </TouchableOpacity>
          </View>
        )}
      ></FlatList>
    </View>
  );
}

const thispage = StyleSheet.create({
  view: {
    width: 350,
    height: 350,
    backgroundColor: "lightgrey",
    margin: 10,
    padding: 8,
    color: "white",
    borderRadius: 14,
    fontSize: 18,
    fontWeight: "500",
  },
});
