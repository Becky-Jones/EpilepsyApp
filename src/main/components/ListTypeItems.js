import React from "react";
import { View, Text, TouchableOpacity, FlatList, Button, StyleSheet } from "react-native";
const styles = require("../pages/stylesheets/styles");

export default function ListTypeItems(props) {
  return (
    <View  style={thispage.view}>
      <FlatList
        data={props.listTypeItems}
        renderItem={({ item }) => (
          <View>
            <TouchableOpacity
              style={{
                padding: 15,
                backgroundColor: "#f8f8f8",
                borderBottomWidth: 1,
                borderColor: "#eee",
                width: 300,
                marginTop: 5,
                justifyContent: "space-between",
                flexDirection: "row",
                display: "flex",
              }}
            >
              <Text> {item.type}</Text>
              <Button
                title="Delete"
                onPress={() => {
                  props.deleteTypeItem(item.id);
                }}
              ></Button>
            </TouchableOpacity>
          </View>
        )}
      ></FlatList>
    </View>
  );
}

const thispage = StyleSheet.create({
  view: {
    width: 350,
    height: 350,
    backgroundColor: "lightgrey",
    margin: 10,
    padding: 8,
    color: "white",
    borderRadius: 14,
    fontSize: 18,
    fontWeight: "500",
  },
});
