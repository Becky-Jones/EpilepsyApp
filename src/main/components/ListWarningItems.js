import React from "react";
import { View, Text, TouchableOpacity, FlatList, Button, StyleSheet } from "react-native";
const styles = require("../pages/stylesheets/styles");

export default function ListWarningItems(props) {
  return (
    <View  style={thispage.view}>
      <FlatList
        data={props.listWarningItems}
        renderItem={({ item }) => (
          <View>
            <TouchableOpacity
              style={{
                padding: 15,
                backgroundColor: "#f8f8f8",
                borderBottomWidth: 1,
                borderColor: "#eee",
                width: 300,
                marginTop: 5,
                justifyContent: "space-between",
                flexDirection: "row",
                display: "flex",
              }}
            >
              <Text> {item.warning}</Text>
              <Button
                title="Delete"
                onPress={() => {
                  props.deleteWarningItem(item.id);
                }}
              ></Button>
            </TouchableOpacity>
          </View>
        )}
      ></FlatList>
    </View>
  );
}

const thispage = StyleSheet.create({
  view: {
    width: 350,
    height: 350,
    backgroundColor: "lightgrey",
    margin: 10,
    padding: 8,
    color: "white",
    borderRadius: 14,
    fontSize: 18,
    fontWeight: "500",
  },
});
