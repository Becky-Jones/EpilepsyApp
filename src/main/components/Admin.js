import { Address } from "./Address";

export class Admin {
  constructor(
    user_id,
    userType,
    firstName,
    surname,
    dob,
    email,
    password,
    gender,
    address,
    city,
    postcode,
    patients
  ) {
    this.userType = userType;
    this.user_id = user_id;
    this.firstName = firstName;
    this.surname = surname;
    this.dob = dob;
    this.email = email;
    this.password = password;
    this.gender = gender;
    this.patients = [];
    this.Address = new Address(address, city, postcode);
  }

  /*****************************
   * GETTERS
   *****************************/

  getName() {
    return this.firstName + " " + this.surname;
  }

  getFirstName() {
    return this.firstName;
  }

  getSurname() {
    return this.surname;
  }

  getDOB() {
    return this.dob;
  }

  getEmail() {
    return this.email;
  }

  getType() {
    return this.userType;
  }

  getId() {
    return this.user_id;
  }

  getPassword() {
    return this.password;
  }

  getGender() {
    return this.gender;
  }

  getPatients() {
    return this.patients;
  }

  getAddress() {
    return this.Address;
  }

  /*****************************
   * SETTERS
   *****************************/

  setFirstName(name) {
    this.firstName = name;
  }

  setSurname(name) {
    this.surname = name;
  }

  setDOB(dob) {
    this.dob = dob;
  }

  setEmail(email) {
    this.email = email;
  }

  setType(type) {
    this.userType = type;
  }

  setPassword(password) {
    this.password = password;
  }

  setGender(gender) {
    this.gender = gender;
  }

  setPatients(patients) {
    console.log("Setting patients with: " + patients.toString());
    this.patients = patients;
  }

}
