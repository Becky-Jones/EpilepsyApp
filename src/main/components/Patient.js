import { Address } from "./Address";
import { SeizureDetails } from "./seizureDetails";

export class Patient {
  constructor(
    user_id,
    userType,
    firstName,
    surname,
    dob,
    email,
    password,
    gender,
    address,
    city,
    postcode,
    practitionerId,
    seizureTriggers,
    seizureTypes,
    seizureFreq,
    yearsSuffered,
    MHIssues
  ) {
    this.userType = userType;
    this.user_id = user_id;
    this.firstName = firstName;
    this.surname = surname;
    this.dob = dob;
    this.email = email;
    this.password = password;
    this.gender = gender;
    this.Address = new Address(address, city, postcode);
    this.practitionerId = practitionerId;
    this.seizureDetails = new SeizureDetails(
      seizureTriggers,
      seizureTypes,
      seizureFreq,
      yearsSuffered,
      MHIssues
    );
  }
  /*****************************
   * GETTERS
   *****************************/

  getName() {
    return this.firstName + " " + this.surname;
  }

  getFirstName() {
    return this.firstName;
  }

  getSurname() {
    return this.surname;
  }

  getDOB() {
    return this.dob;
  }

  getEmail() {
    return this.email;
  }

  getType() {
    return this.userType;
  }

  getId() {
    return this.user_id;
  }

  getPassword() {
    return this.password;
  }

  getGender() {
    return this.gender;
  }

  getAddress() {
    return this.Address;
  }

  getSeizureDetails() {
    return this.seizureDetails;
  }
  
  getPractitionerId() {
    return this.practitionerId;
  }


  /*****************************
   * SETTERS
   *****************************/

  setFirstName(name) {
    this.firstName = name;
  }

  setSurname(name) {
    this.surname = name;
  }

  setDOB(dob) {
    this.dob = dob;
  }

  setEmail(email) {
    this.email = email;
  }

  setType(type) {
    this.userType = type;
  }

  setPassword(password) {
    this.password = password;
  }

  setGender(gender) {
    this.gender = gender;
  }

  setPractitionerId(id) {
    this.practitionerId = id;
  }
}
