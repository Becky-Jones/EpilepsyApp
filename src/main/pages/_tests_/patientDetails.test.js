import React from 'react';
import renderer from 'react-test-renderer';
import { Patient } from '../../components/Patient';

import PatientDetail from '../patientDetails';

describe('<PatientDetail />', () => {
  it('renders correctly for given patient', () => {
    const navigate = jest.fn();
    const patient = {"_id":"619277a7190407ffdfc8a6f8","first_name":"Barry","surname":"Manilow","date_of_birth":"25/10/1989","gender":"Female","email":"neal19@berge.com","password":"UIEInM","address1":"9755 Mill Road","address2":"HUDDERSFIELD","postcode":"HD87 2OC","user_type":"Patient","patient_details":{"seizure_type":["absence seizure"],"years_suffered":4,"seizure_triggers":["lack of sleep"," caffeine"],"seizure_monthly_frequency":1,"mental_health_issues":["Anxiety"],"practitioner_id":"619257063e933ddf5443cb5d"}};
    const user = new Patient("TESTId",
      "Patient",
      "Bob",
      "Bobby",
      "04-08-1998",
      "test@gmail.com",
      "password",
      "male",
      "5 testing lane",
      "liverpool",
      "TR6 0GH",
      "12345678",
      ["Flashing lights"],
      ["Absense Seizure"],
      "4",
      "1",
      ["Anxiety"]);
    const route = { params: { Patient: patient, Movies: "", Patients: "", User: user} }
    const tree = renderer.create(<PatientDetail route = {route} navigation= {{navigate}}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});