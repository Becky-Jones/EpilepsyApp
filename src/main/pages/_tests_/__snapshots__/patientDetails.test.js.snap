// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PatientDetail /> renders correctly for given patient 1`] = `
<RCTScrollView
  collapsable={false}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
>
  <View>
    <RCTScrollView
      collapsable={false}
      onGestureHandlerEvent={[Function]}
      onGestureHandlerStateChange={[Function]}
    >
      <View>
        <View>
          <Text
            style={
              Object {
                "color": "black",
                "fontSize": 20,
                "marginLeft": 100,
                "textAlign": "left",
              }
            }
          >
            Barry
            's Details:
          </Text>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                First Name: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="Barry"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Surname: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="Manilow"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Gender: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="Female"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Postcode: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="HD87 2OC"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Email: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="neal19@berge.com"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 30,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Date of Birth: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="25/10/1989"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 30,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Address: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="9755 Mill Road"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 30,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                City: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="HUDDERSFIELD"
                onChangeText={[Function]}
                placeholderTextColor="black"
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 30,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "white",
                  "borderColor": "black",
                  "borderWidth": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "margin": 10,
                  "padding": 4,
                  "width": undefined,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 4,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "grey",
                      "fontSize": 12,
                      "width": 80,
                    }
                  }
                >
                  Seizure Type(s): 
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flex": 4,
                  }
                }
              >
                <TextInput
                  allowFontScaling={true}
                  defaultValue="absence seizure"
                  onChangeText={[Function]}
                  rejectResponderTermination={true}
                  style={
                    Object {
                      "backgroundColor": "white",
                      "borderRadius": 14,
                      "fontSize": 12,
                      "height": 60,
                      "padding": 8,
                      "width": 150,
                    }
                  }
                  underlineColorAndroid="transparent"
                />
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Years Suffered: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="4"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Seizure Freq: 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="1"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 14,
                    "height": 30,
                    "width": 85,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Seizure Trigger(s) 
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="lack of sleep, caffeine"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 60,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "black",
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "margin": 10,
                "padding": 4,
                "width": undefined,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 12,
                    "width": 80,
                  }
                }
              >
                Mental Health issue(s)
              </Text>
            </View>
            <View
              style={
                Object {
                  "flex": 4,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                defaultValue="Anxiety"
                onChangeText={[Function]}
                rejectResponderTermination={true}
                style={
                  Object {
                    "backgroundColor": "white",
                    "borderRadius": 14,
                    "fontSize": 12,
                    "height": 60,
                    "padding": 8,
                    "width": 150,
                  }
                }
                underlineColorAndroid="transparent"
              />
            </View>
          </View>
        </View>
        <View
          accessibilityRole="button"
          accessibilityState={Object {}}
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {},
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#007AFF",
                    "fontSize": 18,
                    "margin": 8,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Save Changes
            </Text>
          </View>
        </View>
        <View
          accessibilityRole="button"
          accessibilityState={Object {}}
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {},
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#007AFF",
                    "fontSize": 18,
                    "margin": 8,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Remove Patient
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTScrollView>
`;
